change/@ni-nimble-components-46b1fdfb-63b3-4a04-ac71-1124855a4e45.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Pro tip: in your PR description if you link to the issue using words like `resolves #1454` or `fixes #1454` then GitHub will automatically close it when the PR completes. It's handy to do this for issues like bugs or small features that don't require any validation or discussion post-commit.